server/routes/ad.ts

```diff
@@ -1008,6 +1013,8 @@ export default class AdService {
                 ? toFixedNumberForAnomaly(Number.parseFloat(featureData.data))
                 : 0,
             name: featureData.feature_name, 
+            expectedValue: result.anomaly_grade > 0 
+            ? this.getExpectedValue(result._source.expected_values[0].value_list, featureData.feature_id) : featureData.data
```

Comment:
if the anomaly grade is at 0, will the expectedValue not still have some content? Might be misunderstandings whats in featureData.data

Reply:
if the anomaly grade is at 0, there's no expected value. The current value is the expected value

Reply:
will featureData.data be empty than if no expected value, leading to no expected value?